.gitignore

```diff
@@ -7,3 +7,6 @@ terraform.tfstate.backup
 bin
 vendor
 build
+
+.DS_Store
+.idea
```

Comment:
This kind of changes should put in personal git configuration, such as `$HOME/.config/git/ignore`.
Please remove this change.

ref: https://git-scm.com/docs/gitignore

Reply:
Done.

Reply:
Thanks!